source/uwp/Renderer/lib/XamlBuilder.cpp

```diff
@@ -1214,41 +1263,93 @@ namespace AdaptiveCards { namespace Uwp
 
         ABI::AdaptiveCards::Uwp::ImageStyle imageStyle;
         THROW_IF_FAILED(adaptiveImage->get_Style(&imageStyle));
+        ComPtr<IAdaptiveCardResourceResolvers> resourceResolvers;
+        THROW_IF_FAILED(renderContext->get_ResourceResolvers(&resourceResolvers));
 
         ComPtr<IFrameworkElement> frameworkElement;
         if (imageStyle == ImageStyle_Person)
         {
             ComPtr<IEllipse> ellipse = XamlHelpers::CreateXamlClass<IEllipse>(HStringReference(RuntimeClass_Windows_UI_Xaml_Shapes_Ellipse));
-            SetImageOnUIElement(imageUri.Get(), ellipse.Get(), renderContext);
-
-            // Set Auto, None, and Stretch to Stretch_UniformToFill.  An ellipse set to Stretch_Uniform ends up with size 0.
-            if (size == ABI::AdaptiveCards::Uwp::ImageSize::Auto ||
-                size == ABI::AdaptiveCards::Uwp::ImageSize::None ||
-                size == ABI::AdaptiveCards::Uwp::ImageSize::Stretch)
+            SetImageOnUIElement(imageUri.Get(), ellipse.Get(), resourceResolvers.Get());
+
+            ComPtr<IShape> ellipseAsShape;
+            THROW_IF_FAILED(ellipse.As(&ellipseAsShape));
+            // Set Auto, None, and Stretch to Stretch_UniformToFill. An ellipse set to Stretch_Uniform ends up with size 0.
+            if (size == ABI::AdaptiveCards::Uwp::ImageSize::None ||
+                size == ABI::AdaptiveCards::Uwp::ImageSize::Stretch ||
+                size == ABI::AdaptiveCards::Uwp::ImageSize::Auto)
             {
-                ComPtr<IShape> ellipseAsShape;
-                THROW_IF_FAILED(ellipse.As(&ellipseAsShape));
-
                 THROW_IF_FAILED(ellipseAsShape->put_Stretch(Stretch::Stretch_UniformToFill));
             }
```

Comment:
Stretch_UniformToFill is the default value. Since we're not changing this for any size, I'm removing these lines.